aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -41,4 +41,87 @@ spec aptos_framework::block {
 
         aborts_if false;
     }
+
+    /// The caller is aptos_framework.
+    /// The new_epoch_interval must be greater than 0.
+    /// The BlockResource is not under the caller before initializing.
+    /// The Account is not under the caller until the BlockResource is created for the caller.
+    /// Make sure The BlockResource under the caller existed after initializing.
+    /// The number of new events created does not exceed MAX_U64.
+    spec initialize(aptos_framework: &signer, epoch_interval_microsecs: u64) {
+        include Initialize;
+        include New_event_handle;
+    }
+
+    spec schema Initialize {
+        use std::signer;
+        aptos_framework: signer;
+        epoch_interval_microsecs: u64;
+
+        let addr = signer::address_of(aptos_framework);
+        aborts_if addr != @aptos_framework;
+        aborts_if epoch_interval_microsecs <= 0;
+        aborts_if exists<BlockResource>(addr);
+        ensures exists<BlockResource>(addr);
+    }
+
+    spec schema New_event_handle {
+        use std::signer;
+        aptos_framework: signer;
+
+        let addr = signer::address_of(aptos_framework);
+        let account = global<account::Account>(addr);
+        aborts_if !exists<account::Account>(addr);
+        aborts_if account.guid_creation_num + 2 > MAX_U64;
+    }
+
+    /// The caller is @aptos_framework.
+    /// The new_epoch_interval must be greater than 0.
+    /// The BlockResource existed under the @aptos_framework.
+    spec update_epoch_interval_microsecs(
+        aptos_framework: &signer,
+        new_epoch_interval: u64,
+    ) {
+        include Update_epoch_interval_microsecs;
```

Comment:
Not: Why capitalize here? And other places as well

Reply:
Yeah, a convention is that schema names are in camelcase without `_`. I see this is corrected in the newer commit pushed.